scripts/extract_translation_templates.py

```diff
@@ -90,11 +91,36 @@ def check_schema_templates(source_dir, target_dir):
 
 
 if __name__ == "__main__":
+
+    response = requests.get("https://api.github.com/repos/ONSdigital/eq-translations/commits/master").json()
+    commit_sha = response.get('sha')
+
+    def sha_from_pipfile():
+        f = open("Pipfile.lock", "r")
+        file = f.readlines()
+        for i, line in enumerate(file):
+            if '"eq-translations"' in line:
+                ref = re.compile(r"\w{40}")
+                mo = ref.search(file[i + 3])
+                return mo.group()
+        logger.error("eq-translations not found in Pipfile.lock")
+        sys.exit(0)
+
+
+    if sha_from_pipfile() == commit_sha:
+        logger.error("eq-translations up to date")
+
+    else:
+        logger.error('Newer version of eq-translations available, use "pipenv update"')
+        sys.exit(0)
```

Comment:
- What happens if Github is unavailable, or rate limiting? 
- I think the eq-translations check should be in a separate script. 
- We should be doing releases of eq-translations (i.e. update version in setup.py and tag) and using that version for the check.

Reply:
for rare limiting and lost connection can I use "try" and "except" blocks?

Reply:
Seems like, everything from this list has been done, just a tagged release left, thanks👍